### ISSUE TITLE
WebDAV with 502

### ISSUE THREAD

**Issue by AgeBee** [syncany/syncany#578]
Hi,
I get the following error while trying to connect to a remote location via WebDAV:

Connecting to repo for folder /home/harald/temp/Sync ...ERROR

Unable to connect to folder (code: 502)
Testing the remote storage failed.

- Was the connection successful: YES
- Files can be created: YES
- Files can be written to: YES
- The target folder/repo exists: YES

So it seems that all was ok but I get a 'Bad Gateway' error.

I use Linux Mint 17.2. I can connect to the remote location within my browser. So principally the connection works from my computer.

Are there any logfiles I can provide you with ?

Thanks for your help.
Regards
AgeBee

**Comment by atnpgo** [comment]
I am also experiencing the same issue. The remote server is using a certificate issued by Let's Encrypt in case this is relevant.

**Comment by LinuxfarmerHH** [comment]
Got the same problem with web.de as cloud hoster, first i got a certificate to accept with OK, than these appear while contact. The webdav can be mounted by ubuntu filemanagers like nautilus or thunar pretty well through davs.

Connecting to repo for folder /home/eric/WEBdav ...ERROR

Unable to connect to folder (code: 502)
Testing the remote storage failed.

- Was the connection successful: YES
- Files can be created: YES
- Files can be written to: NO
- The target folder/repo exists: YES

**Comment by Bouni** [comment]
Same problem here!

Syncany 0.4.7-alpha on a arch linux machine, remote location is a nextcloud server, running nextcloud 10.
Webdav sync using an Android app works fine, so it seems to be a syncany related issue.

Connecting to repo for folder <redacted> ...ERROR

Unable to connect to folder (code: 502)
Testing the remote storage failed.

- Was the connection successful: YES
- Files can be created: YES
- Files can be written to: NO
- The target folder/repo exists: YES

Any ideas?

**Comment by jeanibarz** [comment]
Same here

![image](https://user-images.githubusercontent.com/3049704/36785856-ab04749c-1c84-11e8-891e-c34a233be2c9.png)

**Comment by Derkades** [comment]
Same issue, nextcloud 15 server. Other webdav clients (keepass, FolderSync for android) work fine

**Comment by theAkito** [comment]
> 
> 
> Same issue, nextcloud 15 server. Other webdav clients (keepass, FolderSync for android) work fine

Sorry man, but nobody is here to fix it, unfortunately.

**Comment by Derkades** [comment]
Yeah I noticed, that's unfortunate. I haven't been able to find a good alternative yet

**Comment by theAkito** [comment]
> 
> 
> Yeah I noticed, that's unfortunate. I haven't been able to find a good alternative yet

https://github.com/borgbackup/borg
